### ISSUE TITLE
Disable seperation of directories and files

### ISSUE THREAD

**Issue by FinlayDaG33k** [aleksey-hoffman/sigma-file-manager#253]
## Feature request:
As the title suggests, it'd be nice to disable the separation of directories and files when viewing them.  
Currently, there is a section that will always list directories at the top followed by a section that will list all files.  
I'd like to have them be shown based solely on my sorting preferences without the split.

**Comment by aleksey-hoffman** [comment]
Yeah I can see how this would be preferable to some users, I can add a setting for that in the next update
